client/src/components/events/CreateEventTypeForm.vue

```diff
@@ -210,14 +220,43 @@ export default {
     },
 
     computed: {
-        eventLinkDashed() {
-            return this.eventName.replace(/\s/g, '-');
+        ...mapGetters('eventType', {
+            getEventTypeForm: eventTypeGetters.GET_EVENT_TYPE_FORM
+        }),
+        data() {
+            return Object.keys(this.form).some(el => !!this.form[el]) ? this.form : this.getEventTypeForm;
```

Comment:
not exactly, okay, if you don't want to create computed for each property, you can spread the data:

```
return {
    ...this.getEventTypeForm,
   ...this.form
}
```